LayoutTests/transitions/svg-text-shadow-transition.html

```diff
@@ -21,17 +21,16 @@
     <script type="text/javascript">
         const expectedValues = [
             // [time, element-id, property, expected-value, tolerance]
-            [0, 'text', 'text-shadow', [-25, -5, 0, 15], 4],
-            [1, 'text', 'text-shadow', [5, 5, 0, 15], 4]
+            [0, 'text', 'text-shadow', [[254, 0, 0], -25, -5, 15], 4],
```

Comment:
This should be 255, not 254. But also, I need to figure out why the tolerance of 4 isn’t close enough to match in WPE tests:
 ```
FAIL - "text-shadow" property for "text" element at 0s expected: 254,0,0,-25,-5,15 but saw: rgb(255, 0, 0) -24.969999px -4.99px 15px
FAIL - "text-shadow" property for "text" element at 1s expected: 0,128,0,5,5,15 but saw: rgb(3, 126, 0) 4.61px 4.87px 15px
```